crates/sui-graphql-rpc/src/types/suins_registration.rs

```diff
@@ -57,6 +65,12 @@ pub(crate) struct SuinsRegistration {
     pub native: NativeSuinsRegistration,
 }
 
+pub(crate) struct NameServiceQuery {
```

Comment:
some doc comments on what this struct is intended for ... and maybe a better name than `NameServiceQuery`? `ParentAndRootNameRecord`?